docs/knowledgebase/integrate/subkey.md

```diff
@@ -101,7 +156,9 @@ The output gives us the following information about our key:
   to write down your key by hand.
 - **Secret Seed** (aka "Private Key" or "Raw Seed") - The minimum necessary information to restore
   the key pair. All other information is calculated from the seed.
-- **Public Key** (aka "Account ID") - The public half of the cryptographic key pair in hexadecimal.
+- **Public Key (hex)** - The public half of the cryptographic key pair in hexadecimal.
+- **Public Key (SS58)** - The public half of the cryptographic key pair in SS58 encoding.
+- **Account ID** - Alias for the Public Key in hexadecimal. 
 - **SS58 Address** (aka "Public Address") - An SS58-encoded address based on the public key.
 
 Currently Subkey supports the following cryptographic key pairs and signing algorithms:
```

Comment:
Maybe this should go because it's stated already at the top.

Reply:
If yes, then I suggest to include links for each:
- sr25519:  Schorr signatures on the Ristretto group
- ed25519: ed25519
- secp256k1: ECDSA signatures on secp256k1

Reply:
included in the first instance